### ISSUE TITLE
How to optimize the parameter

### ISSUE THREAD

**Issue by knightcao** [csong27/membership-inference#2]
Hello, Song.
    I try your code via UCI adult's salary data set and cifar10. I get a result similar with yours for adult's salary data set, however for cifar10, i get the a very low accuracy. I guess maybe some parameter is need optimized, such as batch_size or epochs.    
   So, I wonder if you willing to publish your parameters ?  And another question is that for images, I should replace the Target's and Shadow's models from NN to CNN or other?
    Thank you, I really appreciate your work.

**Comment by csong27** [comment]
Hi, 
On cifar10 you will need a bigger neural nets than the one in the demo code. CNN is a good idea :)

**Comment by knightcao** [comment]
Thank you, happy voteday.

**Comment by knightcao** [comment]
I found under fitting will take place with default parameter in most dataset. I dont know whether properly or not? Thank you.

**Comment by csong27** [comment]
You will have to fit each dataset with its own best hyper-parameters.

**Comment by knightcao** [comment]
Thank you, have a good weekend.
